optimum/intel/openvino/quantization.py

```diff
@@ -336,7 +337,7 @@ def build_from_quantization_config(self, config: OVQuantizationConfigBase) -> OV
                 )
             else:
                 raise Exception
-        elif is_diffusers_available() and isinstance(self.model, OVDiffusionPipeline):
+        elif _is_diffuser_instance(self.model):
```

Comment:
I would leave `is_diffusers_available() and isinstance(self.model, OVDiffusionPipeline):`, notice that this function would fail if OVDiffusionPipeline is not imported beforehand, and it checks for OVDiffusionPipeline and not DiffusionPipeline

Reply:
I see your point. Thanks for the explanation! I suppose the same holds for `_is_sentence_transformer_instance()`?